apps/wallet-mobile/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@yoroi/wallet-mobile",
-  "version": "4.28.1",
+  "version": "4.28.3",
```

Comment:
is this ok?

```suggestion
  "version": "4.28.2",
```

Reply:
There was a `4.28.2` attempt that downgraded the sdk instead of updating the dependencies, since there's was a publish attempt to the play store I wasn't sure if it would interfere, thus skipping that one just in case.